src/contour/TerminalWindow.h

```diff
@@ -25,7 +25,11 @@
 #include <QtCore/QPoint>
 #include <QtCore/QTimer>
 #include <QtGui/QOpenGLExtraFunctions>
-#include <QtGui/QOpenGLWindow>
+#if defined(CONTOUR_BUILD_WITH_QT6)
```

Comment:
```suggestion
#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
```

Reply:
At first I tried this, but it requires including `<QtCore/QtGlobal>` to `ShaderConfig.h`. :)
Is it acceptable?

Reply:
Yes. Damn I saw that too late.